setup-dynamodb-local.mjs

```diff
@@ -0,0 +1,29 @@
+import { mkdir } from 'fs/promises'
+import { existsSync } from 'fs'
+import { execSync } from 'child_process'
+
+const dynamodbDir = 'dynamodb_local_latest'
+const dynamodbArchiveURL =
+  'https://s3.us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz'
+
+// Check if the dynamodbDir directory exists
+async function runSetup() {
+  if (!existsSync(dynamodbDir)) {
+    // If it doesn't exist, create it
+    await mkdir(dynamodbDir, { recursive: true })
+
+    // Download and extract DynamoDB Local
+    try {
+      console.log('Downloading DynamoDB Local...')
+      execSync(`curl -L ${dynamodbArchiveURL} | tar -xz -C ${dynamodbDir}`, {
+        stdio: 'inherit',
+      })
```

Comment:
- [ ] Use web `fetch()` instead of curl. It's built in to Node 18 and newer.
- [ ] Use [node-tar](https://github.com/isaacs/node-tar) instead of calling out to the command line.